app/__tests__/app.test.js

```diff
@@ -450,4 +450,54 @@ describe("PATCH /api/articles/:article_id", () => {
         })
 
     })
-}) 
\ No newline at end of file
+}) 
+
+describe("DELETE /api/comments/:comment_id", () => {
+    describe("-- functionality testing", () => {
+    test("200: responds with a 200 status and the deleted comment with the comment id", () => {
+        return request(app)
+        .delete("/api/comments/1")
+        .expect(200)
```

Comment:
could alternatively do a 204 no content status if you aren't bothered about returning the deleted comment to the client